package.json

```diff
@@ -74,6 +82,7 @@
     ],
     "instrument": false,
     "lines": 100,
+    "reporter": "text-summary",
```

Comment:
@brettz9 what I mean is that this would be the place to change it, since no other test script displays coverage at all

Reply:
I see. But if we are going to keep accommodating both of our requirements, we need an extra one. That's why I had a `no-cov` before and a `test` that did do coverage and `test-cov` which did a more complete report.

That reminds me, are you ok for me to reenable coverage checks, @gajus ?